test/docker-compose.integration-tests.yml

```diff
@@ -13,7 +13,6 @@ services:
       args:
         - CARGO_PROFILE=debug
     command: bash -c "
-      wait-for-it grapl-provision:${WAIT_PORT} --timeout=250 &&
```

Comment:
why is it removed in the integration tests as well?

Reply:
It's redundant this point with the introduction of the `wait-for-local-provision` Make target that's used to bring up the local Grpal environment for both E2E and integration tests.